VideoWeb/VideoWeb.IntegrationTests/VideoWeb.IntegrationTests.csproj

```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="NUnit" Version="3.13.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="UserApi.Client" Version="1.24.3" />
-    <PackageReference Include="VideoApi.Client" Version="1.27.6" />
+    <PackageReference Include="VideoApi.Client" Version="1.27.7-pullrequest0408-0002" />
```

Comment:
appears to be everything pretty sure this needs to be changed to a more stable version

Reply:
Need video api to be merged